### PR TITLE
[Anatolie Novojdion] - Home work

### PR DESCRIPTION
**Pull request by anovojdion**
Java primitives

### CODE REVIEW

File: Anatolie/src/main/resources/Text_01.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello world
```
Comment (VitalinaPetrenco) [inline Anatolie/src/main/resources/Text_01.txt]: keep your project clean: remove unnecessary files

File: Anatolie/src/main/java/AreaOfCircle.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+public class AreaOfCircle {
+    public static void main(String[] args) {
+
+        double radius;
+        radius = 3;
```
Comment (VitalinaPetrenco) [inline Anatolie/src/main/java/AreaOfCircle.java]: Code would looks better with these statements written inline (ex: double radius = 3;)
the same for area variable

File: Anatolie/src/main/java/AreaOfCircle.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+        double Area;
```
Comment (VitalinaPetrenco) [inline Anatolie/src/main/java/AreaOfCircle.java]: Please follow the naming convention (local variable name - first letter lowercase)

File: Anatolie/src/main/java/Month.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+            case 12:
+                System.out.println("December");
+                break;
+            default:
+                System.out.println("Invalid Month");
+        }
+    }
+
+    public static void main(String[] args) {
+        printMonthByNumber(1);
```
Comment (VitalinaPetrenco) [inline Anatolie/src/main/java/Month.java:46]: Good job moving the printMonth logic into its own method.
Keep following this approach in all you further tasks: its always good to have your logic split into separate meaningful methods as it helps to understand your code and makes it reusable.

File: Anatolie/src/main/java/Numbers.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
+public class Numbers {
+    public static void main(String[] agrs) {
+        for (int i = 1; i <= 10; i = i + 1) {
```
Comment (VitalinaPetrenco) [inline Anatolie/src/main/java/Numbers.java]: There is an option to shorten i = i + 1 to i++

File: Anatolie/src/main/java/Text.java [hunk 6]
```diff
@@ -0,0 +1,9 @@
+public class Text {
+        public static void main(String[] args) {
+            String lower_string = "HeLLo";
```
Comment (VitalinaPetrenco) [inline Anatolie/src/main/java/Text.java]: Please follow the naming convention

File: Anatolie/src/main/java/Text.java [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+            String upper_String = "heLLO";
+            if (lower_string != upper_String) {
```
Comment (VitalinaPetrenco) [inline Anatolie/src/main/java/Text.java]: Its not save to use == operator to compare strings
see also https://www.programiz.com/java-programming/examples/compare-strings#:~:text=You%20should%20not%20use%20%3D%3D,and%20not%20the%20object%20itself.

**Comment by venache1** [comment]
Please squash your commits after fixing all messages.

File: Anatolie/src/main/resources/Text_01.txt (hunk 1 above)
Comment (anovojdion) [reply Anatolie/src/main/resources/Text_01.txt]: Done

File: Anatolie/src/main/java/AreaOfCircle.java (hunk 2 above)
Comment (anovojdion) [reply Anatolie/src/main/java/AreaOfCircle.java]: Done

File: Anatolie/src/main/java/AreaOfCircle.java (hunk 3 above)
Comment (anovojdion) [reply Anatolie/src/main/java/AreaOfCircle.java]: Done

File: Anatolie/src/main/java/Numbers.java (hunk 5 above)
Comment (anovojdion) [reply Anatolie/src/main/java/Numbers.java]: Done

File: Anatolie/src/main/java/Text.java (hunk 6 above)
Comment (anovojdion) [reply Anatolie/src/main/java/Text.java]: Done

File: Anatolie/src/main/java/Text.java (hunk 7 above)
Comment (anovojdion) [reply Anatolie/src/main/java/Text.java]: Done

**Comment by venache1** [comment]
image
Yout still have 5 commits. Could you squash them and force push the branch?

File: Anatolie/src/main/java/RegexDateHW.java [hunk 8]
```diff
@@ -0,0 +1,17 @@
+import javax.swing.*;
+
+public class RegexDateHW {
+    public static void main(String[] args) {
+        validateDate("0991-12-31");
+    }
+
+    public static void validateDate(String date) {
+        if (date.matches("^(([1]{1})([9]{1})([0-9]{2})|([2]{1})([0]{1})([0-9]{2}))-((0)([1-9]{1})|(1)([0-2]{1}))-(([0-2]{1})([0-9]{1})|([3]{1})([0-1]{1}))$"))
```
Comment (venache1) [inline Anatolie/src/main/java/RegexDateHW.java]: Could you simplify this regex?

Comment (anovojdion) [reply Anatolie/src/main/java/RegexDateHW.java]: Done

**Review by venache1 — APPROVED** [review]

**Review by Claudia-20 — APPROVED** [review]

**Review by irinazota — APPROVED** [review]

**Pull request merged**